resources/js/types/Mismatch.ts

```diff
@@ -27,6 +27,12 @@ interface Mismatch {
 
 export interface LabelledMismatch extends Mismatch {
     property_label: string,
+    /**
+     * A display label for the mismatch’s wikidata_value;
+     * this may be the label of the referenced item for an item-type property,
+     * or a formatted data value for other properties.
```

Comment:
> for other properties

Right now, only for time properties, right?

Reply:
Yes, though I don’t want to specify this here (to me, “this may be” already implies that the property is optional, without going into detail about when it exists or not).

Reply:
Mh, I find this to be ambiguous enough that I would like to see it changed.

How about this:

```suggestion
     * or a formatted data value for some other properties.
```